src/test/java/org/jabref/logic/integrity/EditionCheckerTest.java

```diff
@@ -0,0 +1,23 @@
+package org.jabref.logic.integrity;
+
+import org.jabref.model.database.BibDatabaseContext;
+import org.junit.jupiter.api.Test;
+
+
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+
+public class EditionCheckerTest {
+     boolean allowIntegerEdition;
+
+    @Test
+    void isFirstCharacterANumber(){
+        var bibDatabaseContextEdition=new BibDatabaseContext();
+        var editionChecker=new EditionChecker(bibDatabaseContextEdition,allowIntegerEdition);
+        var stringWithNumber="0HelloWorld";
+        boolean flag=editionChecker.isFirstCharDigit(stringWithNumber);
```

Comment:
Combine this and the next line to:

```java
assertTrue(editionChecker.isFirstCharDigit(stringWithNumber));`
```

No need for text outputs in the assert statements.

Reply:
Done